doc/requirements.txt

```diff
@@ -10,7 +10,7 @@ m2r2
 numpy
 pygments-github-lexers
 semantic_version==2.10
-scipy<=1.10
+scipy<=1.11.1
```

Comment:
Do we need to keep the pinning? Also please can we update in `setup.py`?

Reply:
_Our_ qchem functions are now compatible with the latest scipy, so we can remove the pinning with respect to that. However, an upgrade in pyscf needs to be released to make the latest pyscf also compatible with the latest scipy. If we change the pinning from `<=1.10`, the pyscf branch of qchem fails. See [here](https://github.com/pyscf/pyscf/issues/1783) for more details.